plugins/catalog-backend-module-gitlab/src/lib/client.ts

```diff
@@ -107,12 +115,92 @@ export class GitLabClient {
           },
           body: JSON.stringify({
             variables: { group: groupPath, endCursor },
-            query: `query($group: ID!, $endCursor: String) {
+            query: `query listDescendantGroups($group: ID!, $endCursor: String) {
+              group(fullPath: $group) {
+                descendantGroups(first: 100, after: $endCursor) {
+                  nodes {
+                    id
+                    name
+                    description
+                    fullPath
+                    parent {
+                      id
+                    }
+                  }
+                  pageInfo {
+                    endCursor
+                    hasNextPage
+                  }
+                }       
+              }
+            }`,
```

Comment:
This way `prettier` has your back
```suggestion
            query: /* GraphQL */ `
              query listDescendantGroups($group: ID!, $endCursor: String) {
                group(fullPath: $group) {
                  descendantGroups(first: 100, after: $endCursor) {
                    nodes {
                      id
                      name
                      description
                      fullPath
                      parent {
                        id
                      }
                    }
                    pageInfo {
                      endCursor
                      hasNextPage
                    }
                  }
                }
              }
            `,
```